testdata/go/v3/memcached-operator/controllers/memcached_controller.go

```diff
@@ -25,6 +25,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/example/memcached-operator/monitoring/metrics"
```

Comment:
this import should be right under the `cachev1alpha1 "github.com/example/memcached-operator/api/v1alpha1"` one

Reply:
Done